camel/typing.py

```diff
@@ -46,10 +49,12 @@ def is_openai(self) -> bool:
             bool: Whether this type of models belongs to OpenAI.
         """
         if self.name in {
-                "GPT_3_5_TURBO",
-                "GPT_3_5_TURBO_16K",
                 "GPT_4",
                 "GPT_4_32k",
+                "GPT_4_TURBO",
+                "GPT_4_TURBO_VISION",
+                "GPT_3_5_TURBO",
+                "GPT_3_5_TURBO_16k",
```

Comment:
Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

Reply:
> Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

done